runtime/queries/jinja_inline/highlights.scm

```diff
@@ -109,12 +105,34 @@
 
 (expression
   "."
-  (expression)+ @variable.member)
+  (expression
+    (binary_expression
+      .
+      (unary_expression
+        (primary_expression
+          (identifier) @variable.member)))))
+
+(expression
+  "."
+  (expression
+    (binary_expression
+      (binary_expression
+        (unary_expression
+          (primary_expression
+            (identifier) @variable.member))))))
 
 (assignment_expression
   "."
   (identifier)+ @variable.member)
 
+; jinja filters
+(binary_expression
+  (binary_operator
+    "|")
+  (unary_expression
+    (primary_expression
+      (identifier) @function.call)))
+
```

Comment:
This pattern surfaced a bug in the jinja_inline parser when it's generated from grammar (which seems not to have happened for quite a while on the grammar repo).

Since this breaks tests on, e.g., tree-sitter, I'll have to remove this until the upstream bug is fixed.